### PR TITLE
COMMERCE-10524 Integration test implementation on the headless-commerce-admin-account-test module

### PR DESCRIPTION
**Pull request by smottal**
https://issues.liferay.com/browse/COMMERCE-10524

### CODE REVIEW

**Comment by smottal** [comment]
ci:test:relevant

File: modules/apps/commerce/headless/headless-commerce/headless-commerce-admin-account-test/src/testIntegration/java/com/liferay/headless/commerce/admin/account/resource/v1_0/test/AccountChannelShippingOptionResourceTest.java [hunk 1]
```diff
@@ -14,16 +14,302 @@
...
+		return new String[] {
+			"accountId", "shippingMethodKey", "shippingOptionId"
+		};
+	}
+
+	@Override
+	protected AccountChannelShippingOption randomAccountChannelShippingOption()
+		throws Exception {
+
+		CommerceChannel localCommerceChannel =
```
Comment (andrea-ale-sbarra) [inline modules/apps/commerce/headless/headless-commerce/headless-commerce-admin-account-test/src/testIntegration/java/com/liferay/headless/commerce/admin/account/resource/v1_0/test/AccountChannelShippingOptionResourceTest.java:130]: try to call this commerceChannel

**Review by andrea-ale-sbarra — COMMENTED** [review]
check if you can change the variable name of the channel

**Comment by smottal** [comment]
ci:test:sf

**Comment by smottal** [comment]
ci:test:relevant

**Pull request closed without merging**